docs/pages/billing/manage.mdx

```diff
@@ -0,0 +1,262 @@
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+**Billing** in the Expo dashboard provides information about your account's currently subscribed plan and monthly usage. It also allows you to manage your plan and billing details.
+
+This guide explains how to manage your account's plans and billing information.
+
+## Manage plans
+
+### View the current plan
+
+<Tabs>
+
+<Tab label="For Organization account">
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu under **Organization settings**.
+- Under **Plan**, you can see the current plan for your account.
+
+For example, an Organization account is subscribed to the Production plan below:
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the current plan for an Organization account."
+  src="/static/images/billing/management/billing-01.png"
+/>
+
+</Tab>
+
+<Tab label="For Personal account">
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu under **Account settings**.
+- Under **Plan**, you can see the current plan for your account.
+
+For example, a Personal account is subscribed to the Production plan below:
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the current plan for a Personal account."
+  src="/static/images/billing/management/billing-02.png"
+/>
+
+</Tab>
+
+</Tabs>
+
+### Upgrade to a new plan
+
+To upgrade to a different plan:
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu in Expo dashboard.
+- Under **Plan**, click **Change Plan** if you are already on a paid plan. If you are on the Free plan, click **Subscribe**. It opens the **Cart** page.
+- Under **Choose a plan**, you can see a list of all available plans. Choose the next plan you want to upgrade and click the **Continue to Review** button at the bottom of this page.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the list of available plans to upgrade."
+  src="/static/images/billing/management/billing-03.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- Next, you are asked to review and confirm your upgrade plan. After reviewing the details on this page, enable the checkbox and click **Continue to Checkout** to open the checkout page.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to upgrade."
+  src="/static/images/billing/management/billing-04.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- On **Checkout**, you are asked to enter your email, card details, and billing address. After adding these details, click **Pay Now** to subscribe to the new plan.
+
+> **Note**: If you are subscribing to [On-demand plan](/billing/plans/#on-demand), you are not charged any fee when you check out. Since this is a usage-based plan, all charges are due at the end of the billing cycle.
+
+### Downgrade a plan
+
+<Tabs>
+
+<Tab label="From Production and Enterprise to On-demand plan">
```

Comment:
We can include `Legacy` with this as well. The process should be the same.

Reply:
We aren't mentioning Legacy anywhere but I guess in this section it could benefit since we have legacy users and it can be helpful for them in this case.